src/fontMetricsData.js

```diff
@@ -1133,17 +1133,17 @@ export default {
         "8901": [-0.05555, 0.44445, 0, 0, 0.27778],
         "8902": [-0.03472, 0.46528, 0, 0, 0.5],
         "8904": [0.005, 0.505, 0, 0, 0.9],
-        "8942": [0.03, 0.9, 0, 0, 0.278],
-        "8943": [-0.19, 0.31, 0, 0, 1.172],
-        "8945": [-0.1, 0.82, 0, 0, 1.282],
+        "8942": [0.03, 0.903, 0, 0, 0.278],
+        "8943": [-0.19, 0.313, 0, 0, 1.172],
+        "8945": [-0.1, 0.823, 0, 0, 1.282],
```

Comment:
These metric changes are presumably why there are subpixel differences in the screenshots.  They seem to be becoming more accurate, though, so presumably an improvement.